src/main/scala/filter2.scala

```diff
@@ -1,19 +1,18 @@
 package era7bio.db.rna16s
 
 import era7bio.db._, csvUtils._, collectionUtils._
-
 import ohnosequences.fastarious.fasta._
 import ohnosequences.statika._
-
 import com.github.tototoshi.csv._
 import better.files._
 
-case object filter2 extends FilterDataFrom(filter1)() {
+// TODO I don't understand what this is trying to do, and how.
```

Comment:
Does this [paragraph](https://github.com/ohnosequences/db.rna16s/blob/f37b457138f78d08c4941c46cb5738486c933179/readme.md#filter2) help?